lib/ansible/modules/storage/netapp/na_elementsw_initiators.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# (c) 2018, NetApp, Inc
```

Comment:
assuming you guys might want to add 2018-2019 @carchi8py ?

Reply:
Let go ahead and merge it as is, and we can come back and change that to 2019 later.

Reply:
👍